javascript/javascript1/week1/EzNamey.js

```diff
@@ -0,0 +1,5 @@
+var firstWords = ["kia", "peugeot", "vw", "toyoto", "volvo", "benz", "nissan", "skoda", "hyundai", "ferrari"];
+var secondWords = ["picanto", "benzin", "gas", "i30", "polo", "gitz", "mini", "opel", "2008", "automat"];
+var randomNumber = Math.floor(Math.random() * 10);
```

Comment:
you can replace 10 with length of an array (i.e *firstWords.length). this gives the value depends on array length.